docs: update getting-started with the strongest version to date

paulRbr opened the pull request:
The getting-started page includes a sentence about the “strongest”
version to date. Since the recent release of 2.1.0 of AsyncAPI spec,
let's update the number here too 🙂

paulRbr committed:
docs: update getting-started with the strongest version to date

The getting-started page includes a sentence about the “strongest”
version to date. Since the recent release of 2.1.0 of AsyncAPI spec,
let's update the number here too 🙂

derberg commented:
@paulRbr oh, we completely forgot about docs 😢 

would you mind updating version in other getting started docs? in all code snippets, etc? I'll handle the interactive tutorial

paulRbr committed:
docs: update all docs pages which mention AsyncAPI version to 2.1.0

Now that 2.1 is out we can show examples with the 2.1.0 version number
to share latest version number in the docs.

paulRbr commented:
> would you mind updating version in other getting started docs? in all code snippets, etc?

Indeed, I forgot that too :grin:. It's done now!

derberg reviewed (approved):
Thanks a lot!!!
